senza/manaus/route53.py

```diff
@@ -281,7 +281,7 @@ def to_alias(self,
         if self.alias_target is not None:
             # Record is already an Alias
```

Comment:
`target_type` is the record type of the new alias record. Currently it's always `A` but in the future it could be  `AAAA`.